I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/events.rpy", line 159, in script
    CENTERED "If you enjoyed this game, please consider following us {color=#8b67b0}@Gloamy Studios{/color} <3"
Exception: Sayer 'CENTERED' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/events.rpy", line 159, in script
    CENTERED "If you enjoyed this game, please consider following us {color=#8b67b0}@Gloamy Studios{/color} <3"
  File "/Applications/renpy-8.3.4-sdk/renpy/ast.py", line 2579, in execute
    Say.execute(self)
  File "/Applications/renpy-8.3.4-sdk/renpy/ast.py", line 581, in execute
    who = eval_who(self.who, self.who_fast)
  File "/Applications/renpy-8.3.4-sdk/renpy/ast.py", line 470, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'CENTERED' is not defined.

macOS-14.6.1-arm64-arm-64bit arm64
Ren'Py 8.3.4.24120703
Forever Yours, The Devil 1.0
Thu Jan 30 21:50:34 2025
